### ISSUE TITLE
UX suggestions

### ISSUE THREAD

**Issue by yfdyh000** [lostindark/DriverStoreExplorer#22]
1. Configure manifest to asking administrator privilege when starting may be more appropriate?
2. No Select none and Reverse selection button, this forces user to restart the program when select old is not satisfied.
3. Size is important for main purpose, and perhaps it should be placed behind the "name", otherwise it is hidden by default for default window size at 1366*768 resolution.
4. "Select old" will select drivers that same version but different publishers, which may bring unexpected results. See the screenshot.
![screenshot](https://user-images.githubusercontent.com/1769875/30245945-ee88fe84-961c-11e7-860d-23f931b9c1c1.png)

5. Provide an option to configure exclusion for size, for example not select items that size <5MB.
6. Perhaps sort the Size in reverse order by default is better.

**Comment by lostindark** [comment]
Thanks for the suggestion!
1. Will add an option for that.
2. There is a context menu "unselect all".
3. I feel naturely the size column should be the last column. I tried on 1366*768 and it still shows. Maybe on your computer the Driver Class is too long so the size is hidden. At this point I still prefer to keep it at the last location.
4. Fixed in check in 6506b31d94726cc3ea571a413b088867ee2c8df5
5. Good suggestion! Will implement that later.
6. This tool is not only used for clean up old drivers. Some may prefer showing drivers in the order of type or provider. For those people trying to delete the biggest driver, they can also click the size column and sort on that.
